reviewer-guide/docs/01-Review/01-identify-changes-in-framework.md

```diff
@@ -0,0 +1,140 @@
+
+For review, a range specifying `>15.3.0 <16.4.1` says the course started at version `15.3.0` and is valid up to all version (excluding) `16.4.1` where a `<=16.4.1` will specify the version including `16.4.1`.
+
+To make versions easier to specify, the squiggly `~` and caret `^` also identify ranges. When `~` is placed before version (e.g., `~15.0.0`), it specifies all versions incremented at the PATCH level. In other words `15.0.1`, `15.0.3`, or `15.0.19` are all valid. In other words, `~15.0.0` equals `>=15.0.0 <15.1.0`.
+
+The caret `^` specifies all version increments at the MINOR and PATCH levels. `^15.0.0` specifies that any of these following versions are valid: `15.0.1`, `15.1.3`, or `15.5.0`. in other words, `^15.0.0` would equal the range `>=15.0.0 <16.0.0`.
+
+### Fine-grain Version Specification
+
+The `||` operator combines versions and version ranges. `^15 || ^16` specifies all MINOR and PATCH version increments in the MAJOR versions `15` and `16`.
+
+A great place to explore semantic versioning is the [npm SemVer calculator](https://semver.npmjs.com/)
+
+### Pre-release Versions
+
+Many projects will have several pre-release versions published before an official version is published. These pre-release versions are subject to change and can be typically ignored when updating lesson code. 
+
+Pre-release tags range from alpha, beta, and rc (release candidate). With the specified order as follows: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+It is good to note that a pre-release is available and what stage they are in as that likely means a MAJOR version update is close to being published and will trigger more reviews.
+
+## Review with SemVer Versions in Mind
+
+You need to identify these **Breaking Changes**​​​​​​​ and keep them in mind when watching through lessons.
+
+Each framework or library will have a CHANGELOG.md that will record the changes of that specific framework. (link to example) Usually found on the respective frameworks github page.
+
+Most frameworks use Semantic Versioning (SemVer) to keep track of updates. We are mostly interested in Major and Minor
+
+Under each new version, there will be a Breaking Changes (or similarly named) section that is where you will see what exactly will break any lesson examples that egghead might have.
+
+## Track Course Dependency Information
+
+egghead is now keeping track of all the relevant dependency versions related to a course
+
+i.e., A course on "State Management with Mobx in React" will track the version range that starts with the course published version and ends with the current version the course example.
+
+Data for this looks like: 
+
+```json
+
+{
+        slug: 'manage-complex-state-in-react-apps-with-mobx',
+        dependencies: {
+          mobx: '>=2.6.3 <=5.0.3',
+          react: '>=15.3.0 <=16.4.1',
```

Comment:
Should update this range if we want to specify ranges with carets (and change the actual data)

Reply:
the problem with `<=16.x.x` is that it negates future versions `16` covers up to `17` and `^16.4.1` is through `17` but suggests that lower versions don't work. I think `15.3 - 16` is more succinct and correct here and `15 - 16` is probably the most accurate AND succinct